react-debounce-input/build.boot

```diff
@@ -0,0 +1,30 @@
+(def +lib-version+ "3.0.0")
+(def +version+ (str +lib-version+ "-0"))
+
+(set-env!
+ :resource-paths #{"resources"}
+ :dependencies   [['cljsjs/boot-cljsjs "0.5.2" :scope "test"]])
```

Comment:
Why?

My scenario.
I use previous version of react in my app.
I added cljsjs/react dep in my first commit in this PR.
But my app began to use new version form cljsjs/react-debounce-input.
And I wrote `[cljsjs/react-debounce-input "3.0.0-0" :exclusions [cljsjs/react]]`.

react-autosuggest [has no dependency from react](https://github.com/cljsjs/packages/blob/master/react-autosuggest/build.boot#L3).